src/SDKs/RecoveryServices.SiteRecovery/AzSdk.RP.props

```diff
@@ -1,7 +1,7 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
   <PropertyGroup>
-    <AzureApiTag>RecoveryServices_2016-08-10;</AzureApiTag>
+    <AzureApiTag>RecoveryServices_2018-01-10;RecoveryServices_2016-08-10;</AzureApiTag>
```

Comment:
@avneeshrai this should only have 1 version.
So basically it should only reflect 2018 API version, assuming that is the newest version.

Reply:
Done